my-app/src/components/Nav.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import logo from "../logo.svg";
+import "../App.css";
```

Comment:
Since we are dealing with global styles it's OK to load them once (which you do in App.jsx). No need to re-import this file in all other places.